azkaban-common/src/main/java/azkaban/imagemgmt/version/VersionSet.java

```diff
@@ -14,35 +14,36 @@
  * the License.
  */
 
-package azkaban.version;
+package azkaban.imagemgmt.version;
 
 /**
  * VersionSet is defined as the set of key-val pairs to uniquely define the execution environment.
- * This version set will have azkaban version, azkaban configuration version, job type versions, etc.
- *
- * Example version set json:
- * {"azConf":"0.0.5","azkaban":"0.0.2","sparkJobType":"0.1.2"}
+ * This version set will have azkaban version, azkaban configuration version, job type versions,
+ * etc.
+ * <p>
+ * Example version set json: {"azConf":"0.0.5","azkaban":"0.0.2","sparkJobType":"0.1.2"}
  */
 public class VersionSet {
```

Comment:
It will be great to have a Map of imagetypes & versions as part of VersionSet. If we have to look up the version for a specific image, parsing json to pull that information is untidy.

Reply:
Not sure if this is the correct place, as this VersionSet object is very similar to a POJO. VersionSetBuilder already accepts a Map to construct this JSON string, hence the Map will be there somewhere which can be leveraged.